src/Logging.chpl

```diff
@@ -85,4 +93,26 @@ module Logging {
             return "%s:%s".format(cd,rawCms(0));        
         }
     }
+    
+    /*
+     * Returns the LogLevel enum corresponding to the supplied log
+     * level string.
+     */
+    proc getLogLevel(level : string) : LogLevel throws {
```

Comment:
I'm not seeing this used anywhere. I also think that casting strings to enums is a built-in feature so you might not need this anyways:

```chpl
enum LogLevel {INFO, DEBUG}
config const level = "INFO";
if level:LogLevel == LogLevel.INFO then writeln("level=info");
```

Reply:
@ronawho Ah gotcha. I added this with the thought that eventually we may have a admin feature where a user could pass in a log level string like "INFO" and then have a *Msg function set the log level for the corresponding module. But yeah, good idea, since we're passing in a CLI string like LogLevel.INFO that is getting converted to the enum.

Reply:
@ronawho removed the unneeded Logging module code.